core/domain/state_domain.py

```diff
@@ -918,8 +942,933 @@ def get_all_outcomes(self) -> List[Outcome]:
             outcomes.append(self.default_outcome)
         return outcomes
 
+    def _validate_continue_input(self) -> None:
+        """Validates the Continue interaction.
+
+        Raises:
+            ValidationError. Text value is empty or is more than 20 characters.
+            ValidationError. Answer group is present.
+        """
+        button_text = self.customization_args['buttonText'].value
```

Comment:
Can be added via schema!

Reply:
I'm a bit confused on how should I handle the validation for the `custom` type objects. Do I have a validation for the complete object and was trying to find out how will the value be passed as.

Reply:
Maybe something like this (has_expected_subtitled_content_length(min_length, max_length)): https://github.com/oppia/oppia/blob/953dc2b1d39f604bca41db8540b865140e14a808/extensions/interactions/interaction_specs.json#L446-L449

Reply:
Done